deps/v8/src/api.cc

```
@@ -939,6 +939,15 @@ void Template::Set(v8::Local<Name> name, v8::Local<Data> value,
   i::Isolate* isolate = templ->GetIsolate();
   ENTER_V8(isolate);
   i::HandleScope scope(isolate);
+  auto value_obj = Utils::OpenHandle(*value);
```

Perhaps `TODO: Throw Error instead of warning in v7.0`?